client/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release Notes
 
+## 1.4.0 (YYYY-MM-DD)
+
+- [BETA] Local variables and arguments are suggested when using autocompletion in functions.
+  - For loops local variables as well as local variables declared using the arrow operator (`--> local […]`) are not handled yet.
```

Comment:
I had a hard time reading this, mostly because I thought this initial “for” was the “real” English “for”. Also not that English guys tend to use “…” without the square brackets.

```suggestion
  - Variables declared within “for” loops, as well as those declared using the arrow operator (`--> local …`), are not handled yet.
```